src/pudl/extract/dbf.py

```diff
@@ -21,6 +22,13 @@
 
 logger = pudl.logging_helpers.get_logger(__name__)
 
+warnings.filterwarnings(
+    action="once",
+    module="pudl.extract.dbf",
+    category=FutureWarning,
+    message="The behavior of DataFrame concatenation with empty or all-NA entries is deprecated",
+)
+
```

Comment:
Another repetitive warning that we don't need to see a bazillion times.  But this filter is not working.